src/spyglass/position_linearization/v1/linearization.py

```diff
@@ -0,0 +1,304 @@
+    Used for linearizing position."""
+
+    definition = """
+    track_graph_name : varchar(80)
+    ----
+    environment : varchar(80)    # Type of Environment
+    node_positions : blob  # 2D position of track_graph nodes, shape (n_nodes, 2)
+    edges: blob                  # shape (n_edges, 2)
+    linear_edge_order : blob  # order of track graph edges in the linear space, shape (n_edges, 2)
+    linear_edge_spacing : blob  # amount of space between edges in the linear space, shape (n_edges,)
+    """
+
+    def get_networkx_track_graph(self, track_graph_parameters=None):
+        if track_graph_parameters is None:
+            track_graph_parameters = self.fetch1()
+        return make_track_graph(
+            node_positions=track_graph_parameters["node_positions"],
+            edges=track_graph_parameters["edges"],
+        )
+
+    def plot_track_graph(self, ax=None, draw_edge_labels=False, **kwds):
+        """Plot the track graph in 2D position space."""
+        track_graph = self.get_networkx_track_graph()
+        plot_track_graph(
+            track_graph, ax=ax, draw_edge_labels=draw_edge_labels, **kwds
+        )
+
+    def plot_track_graph_as_1D(
+        self,
+        ax=None,
+        axis="x",
+        other_axis_start=0.0,
+        draw_edge_labels=False,
+        node_size=300,
+        node_color="#1f77b4",
+    ):
+        """Plot the track graph in 1D to see how the linearization is set up."""
+        track_graph_parameters = self.fetch1()
+        track_graph = self.get_networkx_track_graph(
+            track_graph_parameters=track_graph_parameters
+        )
+        plot_graph_as_1D(
+            track_graph,
+            edge_order=track_graph_parameters["linear_edge_order"],
+            edge_spacing=track_graph_parameters["linear_edge_spacing"],
+            ax=ax,
+            axis=axis,
+            other_axis_start=other_axis_start,
+            draw_edge_labels=draw_edge_labels,
+            node_size=node_size,
+            node_color=node_color,
+        )
+
+
+@schema
+class LinearizationSelection(dj.Lookup):
+    definition = """
+    -> PositionOutput
+    -> TrackGraph
+    -> LinearizationParameters
+    ---
+    """
+
+
+@schema
+class LinearizedPositionV1(dj.Computed):
+    """Linearized position for a given interval"""
+
+    definition = """
+    -> LinearizationSelection
+    ---
+    -> AnalysisNwbfile
+    linearized_position_object_id : varchar(40)
+    """
+
+    def make(self, key):
+        print(f"Computing linear position for: {key}")
+
+        key["analysis_file_name"] = AnalysisNwbfile().create(
+            key["nwb_file_name"]
+        )
+
+        position_nwb = (PositionOutput & key).fetch_nwb()[0]
+
+        position = np.asarray(
+            position_nwb["head_position"].get_spatial_series().data
+        )
+        time = np.asarray(
+            position_nwb["head_position"].get_spatial_series().timestamps
+        )
```

Comment:
I believe the update to the position pipeline renamed the NWB objects for position to just `position` with no `head_` prefix.

```suggestion
        position = np.asarray(
            position_nwb["position"].get_spatial_series().data
        )
        time = np.asarray(
            position_nwb["position"].get_spatial_series().timestamps
        )
```